src/components/custom/DatePicker/components/DateRangePicker.tsx

```
@@ -0,0 +1,115 @@
+ */
+export const DateRangePicker: React.FC<DatePickerProps> = ({
+  startDate,
+  endDate,
+  onChange,
+  selectedPreset,
+  className,
+  onCancel,
+  onSave,
+  isOpen,
+}) => {
+  const handleRangeChange = (ranges: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)